scargo/commands/flash.py

```diff
@@ -92,7 +101,8 @@ def flash_stm32(config: Config, flash_profile: str = "Debug") -> None:
         logger.error("Flash start address not found in lock file")
         logger.info("Define flash-start in scargo.toml under stm32 section")
     else:
-        subprocess.check_call(["st-flash", "erase"])
+        if erase_memory:
```

Comment:
it wont be better to buildup a list of options in separate variable, and then add it at the end to st-link? will be flexible for future options, something like this:
list = ["st-flash"]
list.extend[option_list]
subprocess.check_call(list)

Reply:
Also I think you don't understand this change. By default we call `st-flash erase` before the actual flashing. This option suppresses this first command. Your proposition does not replace it in any way.